src/restconfig/src/main/java/org/geoserver/rest/catalog/StyleController.java

```diff
@@ -225,7 +225,7 @@ private UriComponents getUriComponents(String name, String workspace,
         return uriComponents;
     }
 
-    @GetMapping(path = {"/styles/{styleName}", "/workspaces/{workspaceName}/styles/{styleName}"})
+    @GetMapping(path = {"/styles/{styleName}", "/workspaces/{workspaceName}/styles/{styleName}"}, produces = {MediaType.ALL_VALUE})
```

Comment:
Again, this should be handled by the standard XStream writers. There is a separate GetMapping for Style output. See styleSLDGet bellow - this is probably where the fix should be applied.

@jodygarnett - You picked up this from @smithkm at the end of the sprint - anything else you can add here? Do you remember how non-SLD styles were handled; it looks like styleSLDGet does not handle them.